Fix proximity and distance fade alpha blend warning, move a GIF (#4383)
godotengine/godot-docs: tutorials/3d/standard_material_3d.rst

.. image:: img/spatial_material25.png

Proximity and Distance Fade

> ----------------------------

Reviewer: ```suggestion
---------------------------
```

Godot allows materials to fade by proximity to each other as well as depending
on the distance from the viewer. Proximity fade is useful for effects such as
soft particles or a mass of water with a smooth blending to the shores.

.. image:: img/spatial_material_proxfade.gif

Distance fade is useful for light shafts or indicators that are only present
after a given distance.

> Keep in mind enabling proximity fade and distance fade with pixel alpha mode

Reviewer: ```suggestion
Keep in mind enabling proximity fade and distance fade with **Pixel Alpha** mode
```

> Keep in mind enabling proximity fade and distance fade with **Pixel Alpha** mode
> enables alpha blending. Alpha blending is more performance intensive, and doesn't
> allow for depth mapping so the mesh can't have a shadow. So abusing them for an
> entire scene is usually not a good idea.

Reviewer: ```suggestion
Keep in mind enabling proximity fade and distance fade with **Pixel Alpha** mode
enables alpha blending. Alpha blending is more GPU-intensive and can cause transparency sorting issues. It also disables many material features such as the ability to cast shadows.
To hide a character when they get too close to the camera, consider using **Pixel Dither** or better, **Object Dither** (which is faster).
```

> To hide a character when they get too close to the camera, consider using **Pixel Dither** or better, **Object Dither** (which is faster).

Reviewer: ```suggestion
To hide a character when they get too close to the camera, consider using **Pixel Dither** or better, **Object Dither** (which is even faster than **Pixel Dither**).
```

.. image:: img/spatial_material25.png

Proximity and Distance Fade
---------------------------

Godot allows materials to fade by proximity to each other as well as depending
on the distance from the viewer. Proximity fade is useful for effects such as
soft particles or a mass of water with a smooth blending to the shores.

.. image:: img/spatial_material_proxfade.gif

Distance fade is useful for light shafts or indicators that are only present
after a given distance.

> Keep in mind enabling proximity fade and distance fade with **Pixel Alpha** mode
> enables alpha blending. Alpha blending is more GPU-intensive and can cause transparency sorting issues.
> Alpha blending also disables many material features such as the ability to cast shadows.
> To hide a character when they get too close to the camera, consider using
> **Pixel Dither** or better, **Object Dither** (which is even faster than
> **Pixel Dither**).

Reviewer: ```suggestion
Keep in mind enabling proximity fade or distance fade with **Pixel Alpha** mode
enables alpha blending. Alpha blending is more GPU-intensive and can cause
transparency sorting issues. Alpha blending also disables many material
features such as the ability to cast shadows.
To hide a character when they get too close to the camera, consider using
**Pixel Dither** or better, **Object Dither** (which is even faster than
**Pixel Dither**).
```
